src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/DetailsVA.jsx

```diff
@@ -87,11 +87,7 @@ export default function DetailsVA({ appointment, facilityData }) {
       <PrintLink appointment={appointment} />
       {isAppointmentCancellable && <CancelLink appointment={appointment} />}
       {!isAppointmentCancellable && (
-        <NoOnlineCancelAlert
-          appointment={appointment}
-          facility={facility}
-          isCompAndPenAppointment={isCompAndPenAppointment}
-        />
+        <NoOnlineCancelAlert appointment={appointment} facility={facility} />
```

Comment:
Eliminated the third parameter since `isCompAndPenAppointment` can be extracted from the `appointment.vaos` object